locale/fr/get-involved/index.md

```diff
@@ -19,7 +19,7 @@ layout: contribute.hbs
 
 * [Node.js Collection](https://medium.com/the-node-js-collection) est une liste de contenus maintenue par la communauté sur Medium (en anglais).
 
-* La [Community Committee](https://github.com/nodejs/community-committee) s'agit d'un comité de haut niveau de la Fondation Node.js axé sur les efforts communautaires.
+* La [Community Committee](https://github.com/nodejs/community-committee) est un comité de haut niveau de la Fondation Node.js axé sur les efforts communautaires.
```

Comment:
Probablement plus juste de passer l'article au masculin.

Reply:
Si le générique de _Community Comittee_ est _Committee_ et que c'est traduit en "comité", effectivement le masculin s'applique — si c'est _Community_, c'est traduit en "communauté" et le genre féminin s'applique.

Reply:
Si je ma plante pas, les termes etrangers sont toujours au neutre (masculin). On dit "un GameBoy" ;p

Reply:
Dans ce cas, faudrait-il l'écrire en français (donc écrire "Le comité communautaire") ?
Après, le déterminant ne m'a pas plus choqué que ça.

Reply:
Au pire, on fait semblant de pas l'avoir vu sur cette itération

Reply:
Je considère donc cette revue comme résolue.